src/main/java/appeng/tile/crafting/TileMolecularAssembler.java

```diff
@@ -338,58 +342,31 @@ public void updateSetting( final IConfigManager manager, final Enum settingName,
 	}
 
 	@Override
-	public IInventory getInternalInventory()
+	public IItemHandlerModifiable getInternalInventory()
 	{
-		return this.inv;
+		return this.internalInv;
 	}
 
 	@Override
-	public int getInventoryStackLimit()
+	protected IItemHandler getItemHandlerForSide(EnumFacing side)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)